syntax/import.go

```diff
@@ -38,6 +55,33 @@ func importLocalFile() rel.Value {
 	return importLocalFileVar
 }
 
+func findRootFromModule(modulePath string, fs afero.Fs) (string, error) {
```

Comment:
no need to use the afero.Fs here, use the os/file/path api directly please

Reply:
A big part of the reason for introducing afero was testability. Is that not applicable here?

Reply:
@marcelocantos make sense, afero can avoid file I/O in unit tests. The latest version is using native path api, do you want me to change it back? Otherwise, I'll merge this pull request if it looks good to you.